packages/grid/x-data-grid-premium/package.json

```
@@ -46,9 +46,9 @@
     "@babel/runtime": "^7.23.7",
     "@mui/system": "^5.15.2",
     "@mui/utils": "^5.15.2",
-    "@mui/x-data-grid": "7.0.0-alpha.7",
-    "@mui/x-data-grid-pro": "7.0.0-alpha.7",
-    "@mui/x-license-pro": "7.0.0-alpha.1",
+    "@mui/x-data-grid": "7.0.0-alpha.8",
+    "@mui/x-data-grid-pro": "7.0.0-alpha.8",
+    "@mui/x-license-pro": "7.0.0-alpha.2",
```

We usually sync versions of all the packages when releasing.
```suggestion
    "@mui/x-license-pro": "7.0.0-alpha.8",
```